### ISSUE TITLE
Doesn't function when plugins are in subfolders

### ISSUE THREAD

**Issue by hollyfromthenexus** [mrpedrobraga/godot-plugin-refresher-4.x#3]
# Description
As of https://github.com/godotengine/godot/pull/43734, plugins can be located in subfolders of subfolders inside of the `/addons` directory. It looks like this plugin does not support this functionality.

# To Reproduce
1. Create a plugin at `/addons/example_suite/sub_plugin`.
![image](https://github.com/mrpedrobraga/godot-plugin-refresher-4.x/assets/1341063/8c3d961b-0633-4e2e-9408-5ec766daafaa)
2. Enable in Project Settings -> Plugins
![image](https://github.com/mrpedrobraga/godot-plugin-refresher-4.x/assets/1341063/7c087303-0208-494e-98a9-6c4e1e827fa6)
3. Select with the Plugin Refresher dropdown
![image](https://github.com/mrpedrobraga/godot-plugin-refresher-4.x/assets/1341063/0a404e99-d38c-44de-a2e0-79903a65de15)
4. Checking the toggle switch produces this warning
![image](https://github.com/mrpedrobraga/godot-plugin-refresher-4.x/assets/1341063/2f02baa4-3cbf-44af-bcbb-41c0e06cff55)
```
editor/editor_node.cpp:3320 - Addon 'res://addons/sub_plugin/plugin.cfg' failed to load. No directory found. Removing from enabled plugins.
```
5. Unchecking produces this error
![image](https://github.com/mrpedrobraga/godot-plugin-refresher-4.x/assets/1341063/26bff043-ce9d-4878-a336-0a84e927ec88)
```
editor/editor_node.cpp:3305 - Condition "!p_enabled && !addon_name_to_plugin.has(addon_path)" is true.
```

**Comment by hollyfromthenexus** [comment]
Note how in the warning, it's trying to load the addon from `res://addons/sub_plugin/plugin.cfg`. This ignores that it's actually in a subfolder... the correct path would be `res://addons/example_suite/sub_plugin/plugin.cfg`.

**Comment by hollyfromthenexus** [comment]
Oh huh, looks like this might get fixed in @coderbloke's https://github.com/mrpedrobraga/godot-plugin-refresher-4.x/pull/2 (specifically https://github.com/mrpedrobraga/godot-plugin-refresher-4.x/pull/2/commits/85b2f98225d04e483f9a90b1d702f597ab62d055).

**Comment by mrpedrobraga** [comment]
@Iseeicy I'll check that PR later today, it seems to change a lot of stuff. If so I'll merge it, and close the issue.

**Comment by mrpedrobraga** [comment]
Closing, as it was resolved by #2 .

**Comment by hollyfromthenexus** [comment]
tysm for the fast review and response! Above and beyond <3
